skills/csharp/experimental/musicskill/Responses/ControlSettings/ControlSettingsResponses.cs

```diff
@@ -0,0 +1,18 @@
+﻿// https://docs.microsoft.com/en-us/visualstudio/modeling/t4-include-directive?view=vs-2017
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace MusicSkill.Responses.ControlSettings
+{
+    /// <summary>
+    /// Contains bot responses.
+    /// </summary>
+    public class ControlSettingsResponses
```

Comment:
static?

Reply:
All the skills use non-static class for the Responses.cs now, so I think it is OK to keep it. If we need to change response.cs to static, we can open another issue to track it.